protocols/delegate/contracts/Delegate.sol

```diff
@@ -131,9 +131,9 @@ contract Peer is IPeer, Ownable {
     * @notice Set a Trading Rule
```

Comment:
revert message 7 lines above this needs updating
`PEER_CONTRACT_OWNER_REQUIRED` -> `DELEGATE_CONTRACT_OWNER_REQUIRED`

Sorry github won't let me comment on the correct line

Reply:
Line 124